source/gent/tier1_hortense.rst

```diff
@@ -151,6 +176,28 @@ Project storage
   cd $VSC_SCRATCH_PROJECTS_BASE/<yourprojectnamehere>
 
 
+Read only filesystem
+********************
+
+The performance of the filee system is not always optimal in case of small files often read without
+writing them (software stack, certain input files, for example ones for DNA sequencing). 
+For those files we defined a read-only partition (mirroring the whole network file system). 
+To access this read-only partition you have to simply suffix the path with ``/readonly``, 
+for example use ``/readonly/$VSC_SCRATCH_PROJECT_BASE`` instead of ``$VSC_SCRATCH_PROJECT_BASE``.
+Please be aware that we only update the read-only partition every half an hour on the login noodes,
+and on the worker nodes every times when a job starts. For exmaple, if you place a file, for example,
+to ``$VSC_SCRATCH_PROJECT_BASE/<project>/<file>`` on the login node, it would only appear at 
+``/readonly/$VSC_SCRATCH_PROJECT_BASE/<project>/<file>`` up to 30 minutes later, or on the worker nodes
+if a job starts. Please note that we do not update the read-only partition during a job, so nothing what you 
+write during your simulation will appear there. 
+
+We do advise to use the read-only partition whenever is possible, as it can seriouly speed up simulations.
```

Comment:
remove `is`
partition -> mount
seriouly -> seriously

it does not speed up anything, it causes no weird sideeffects that might affect the speed like regular scratch cuurently does.